_build/templates/default/sass/login.scss

```diff
@@ -217,6 +217,36 @@ input[type="checkbox"] {
   }
 }
 
+#modx-login-btn {
+  border: 1px solid transparent;
+  &:hover {
+    background-color: $white;
+    color: $primary2;
+    box-shadow: none;
+    border: 1px solid $primary2;
+    transition: color .2s ease-out,
+    background-color .2s ease-out,
+    border .2s ease-out;
+  }
+  &:focus{
+    box-shadow: none;
+  }
+}
+
+button#modx-fl-btn {
+  border: 1px solid transparent;
+  &:hover {
+    background-color: $white;
+    color: $primary1;
```

Comment:
Buttons look the same, but here color $primary1 when in login button $primary2. Why? I guess it should be the same for both buttons.